Dockerfile

```diff
@@ -78,9 +78,9 @@ RUN curl -O https://bootstrap.pypa.io/get-pip.py && \
     rm get-pip.py
 
 RUN pip3 --no-cache-dir install \
-            'notebook==5.0.0' \
-            'jupyterhub==0.7.2' \
-            'jupyterlab==0.24.1' \
+            'notebook==5.4.0' \
+            'jupyterhub==0.8.1' \
+            'jupyterlab==0.31.12' \
```

Comment:
Is jupyterlab needed?

Reply:
When we added this was because I was seeing errors/warning during startup. And the Jupyter team does the same when creating their images.